core/shared/src/main/scala/fs2/concurrent/Channel.scala

```diff
@@ -115,7 +115,7 @@ sealed trait Channel[F[_], A] {
     *
     * No-op if the channel is closed, see [[close]] for further info.
     */
-  def sendAndClose(a: A): F[Either[Channel.Closed, Unit]]
+  def closeWithElement(a: A): F[Either[Channel.Closed, Unit]]
```

Comment:
I guess we should update the scaladoc, something like "Gracefully closes this channel with a final element. This method will never block."